configs/testdata/valkyrie_config.minimal.yml

```diff
@@ -15,14 +15,22 @@ logging: {}
 #    # max_backups: 3                  # the maximum number of old log files to retain
 #    # compress: false                 # determines if the rotated log files should be compressed using gzip
 
-# tracing configuration, see https://opentelemetry.io/docs/concepts/what-is-opentelemetry/
+# telemetry configuration, see https://opentelemetry.io/docs/concepts/what-is-opentelemetry/
+# configures general telemetry metadata
+telemetry: {}
```

Comment:
It makes sense if they share some parameters, which would need to be duplicated otherwise. If not, it seems more readable to keep them separate on top level.